Packs/CortexXDR/IncidentFields/incidentfields-CommandLine_Verdict.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "incident_commandlineverdict",
+    "version": -1,
+    "modified": "2023-09-04T13:29:12.604996864Z",
+    "name": "Command Line Verdict",
+    "ownerOnly": false,
+    "description": "the verdict of the command line analysis",
```

Comment:
Add this validation to the pack ignore IF113
and Fix the other validation (`unsearchable` to true)